build/build-dt-report-resources.js

```diff
@@ -33,6 +33,8 @@ writeFile('report.js', htmlReportAssets.REPORT_JAVASCRIPT);
 writeFile('report.css', htmlReportAssets.REPORT_CSS);
 writeFile('template.html', htmlReportAssets.REPORT_TEMPLATE);
 writeFile('templates.html', htmlReportAssets.REPORT_TEMPLATES);
+writeFile('report.d.ts', 'export {}');
+writeFile('report-generator.d.ts', 'export {}');
```

Comment:
Not important for this PR, but FWIW we can make these now. `emitDeclarationOnly` works for js now, and the report side of types are a lot more cleanly separated than the rest of LH. I don't know if it'll actually help with any DevTools integration, though.